google-cloud-clients/google-cloud-tasks/README.md

```diff
@@ -99,4 +99,4 @@ Apache 2.0 - See [LICENSE] for more information.
 [cloud-platform]: https://cloud.google.com/
 [cloud-tasks]: https://cloud.google.com/tasks
 [tasks]: https://cloud.google.com/tasks/docs
-[tasks]: https://googlecloudplatform.github.io/google-cloud-java/google-cloud-clients/apidocs/index.html?com/google/cloud/tasks/v2beta2/package-summary.html
+[tasks-client]: https://googlecloudplatform.github.io/google-cloud-java/google-cloud-clients/apidocs/index.html?com/google/cloud/tasks/v2beta2/package-summary.html
```

Comment:
To follow the existing pattern for other clients, this should be named `tasks-client-lib-docs`

Reply:
done